src/accountsdb/db.zig

```diff
@@ -275,25 +275,26 @@ pub const AccountsDB = struct {
         /// needs to be a thread-safe allocator
         allocator: std.mem.Allocator,
         /// Must have been allocated with `self.allocator`.
-        combined_manifest: *FullAndIncrementalManifest,
+        combined_manifest: FullAndIncrementalManifest,
```

Comment:
probs should rename this param to something similar to the struct name (since its not collapsed/combined)

Reply:
How about "manifest_pair"? Calling it "full_and_incremental_manifest" everywhere gets pretty long and overflows the 100-line length limit in a lot of places, and imo makes the code harder to follow than a more compressed name like the former

Reply:
(could also consider calling the type itself "ManifestPair", lmk wyt)

Reply:
b25e2e31dba827f6809c790866d4c2c3cc288920